recipes/thriftpy2/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "thriftpy2" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 419312421f58e028f8238735f4d99808c7348e5a3f3e7a19e1d9795c98f177ac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
```

Looks like you need a compiler:
```suggestion
requirements:
  build:
    - {{ compiler('c') }}
```